robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -180,27 +180,64 @@ def populate_pause_modal(self,**kwargs):
                   self.selenium.click_element(selection_value)
               else:
                   self.builtin.log(f"Element {key} not present")
-           else:
-              checkbox =  npsp_lex_locators["erd"]["pause_date_checkbox"].format(value)
-              self.selenium.click_element(checkbox)
+           elif key in ("Date"):
```

Comment:
It would be good to add an else statement here with an exception. Otherwise your keyword will pass without processing that key and might fail at a later point